packages/react-vapor/src/components/form/FormGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+export interface IFormGroupProps {
+    title: string;
+    description?: string;
+    className?: string | string[];
+    children?: React.ReactNode;
+}
+
+export const FormGroup = ({children, title, description, className}: IFormGroupProps) => (
```

Comment:
`export const FormGroup: React.FunctionComponent<IFormGroupProps> = ({children, title, description, className}) => (`

Will allow you to remove `children` from `IFormGroupProps`